tf_patches/cpu_dynamic_shape.diff

```diff
@@ -0,0 +1,61 @@
+# Required for dynamic shape on TFRT:CPU. Remove when we sync past https://github.com/tensorflow/tensorflow/commit/7f045484290f57ec3dc91877af77de39275a4f78
+diff --git a/tensorflow/compiler/xla/pjrt/tfrt_cpu_pjrt_client.cc b/tensorflow/compiler/xla/pjrt/tfrt_cpu_pjrt_client.cc
+index f23bbc13e8c..25f68ffab74 100644
+--- a/tensorflow/compiler/xla/pjrt/tfrt_cpu_pjrt_client.cc
++++ b/tensorflow/compiler/xla/pjrt/tfrt_cpu_pjrt_client.cc
+@@ -1075,18 +1075,25 @@ PjRtFuture<Status> TfrtCpuBuffer::ToLiteral(MutableLiteralBase* literal) {
+
+   bool should_sync_copy = device_buffer_wait_avs.empty() &&
+                           literal->size_bytes() < kSmallDataTransferByteSize;
++  StatusOr<Shape> device_shape = logical_on_device_shape();
```

Comment:
Do we plan to merge this fix on upstream PJRT directly?

Reply:
This is a copy of the linked commit from TensorFlow. I have it patched here temporarily until we update again.

Reply:
SG, please communicate with whoever is doing next pin update so they can delete this patch.